solidus_dev_support.gemspec

```diff
@@ -46,5 +46,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rubocop-rails', '~> 2.3'
   spec.add_dependency 'rubocop-rspec', '~> 2.0'
   spec.add_dependency 'solidus_core', ['>= 2.0', '< 5']
-  spec.add_dependency 'webdrivers', '>= 4.4'
+  spec.add_dependency 'selenium-webdriver', '~> 4.11'
```

Comment:
Our linting is complaining that this line should be one higher to keep this list alphabetical. ❤️